src/utils/link-interceptor.js

```diff
@@ -25,6 +25,11 @@ export const interceptLinks = event => {
     return true;
   }
 
+  const newTab = event.ctrlKey || event.metaKey;
+  if (newTab) {
+    return true; // will open in new tab b/c we dont eat the event
```

Comment:
@samouri I think this comment and the naming could be a bit clearer here, especially since the link interceptor itself doesn't know about new tabs…

if we can rename and update the comments I think the PR will be ready. here's an idea

```js
// we don't want to interfere with the click
// if the user has specifically overwritten the
// normal behavior already by holding down
// one of the modifier keys.
if ( event.ctrlKey || event.metaKey ) {
	return true;
}
```